### PR TITLE
AF-2404 : Package dropdown button is not visible (slowly appears) when creating an asset

### PR DESCRIPTION
**Pull request by akumar074**
* Changed KieEnumSelectElement implementations to be used as Element

 * Fixed Tests

Ensemble:
kie-wb-common [#3105](https://github.com/kiegroup/kie-wb-common/pull/3105)

Deployable WAR link: https://drive.google.com/open?id=11j9krqVd5xgBhoGw_sMBxAkGICkbzYTj

### CODE REVIEW

**Comment by akumar074** [comment]
jenkins retest this please

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/ObjectItemView.java [hunk 1]
```diff
@@ -42,7 +44,7 @@
 
     @Inject
     @DataField("resolvers")
-    private HTMLDivElement resolversContainer;
+    private KieEnumSelectElement<Resolver> resolversContainer;
```
Comment (pefernan) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/ObjectItemView.java]: Same here

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.java [hunk 2]
```diff
@@ -46,7 +48,7 @@
 
     @Inject
     @DataField("resolvers")
-    private HTMLDivElement resolversContainer;
+    private KieEnumSelectElement<Resolver> resolversContainer;
```
Comment (pefernan) [inline jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.java]: Please, give a proper name. `resolversSelect` might be a good name.

Comment (akumar074) [reply jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.java]: Thanks for the review Pere, I have renamed the fields.

File: jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/ObjectItemView.java (hunk 1 above)
Comment (akumar074) [reply jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/ObjectItemView.java]: Done

**Pull request merged**